api/swagger-spec/v1.json

```diff
@@ -20737,8 +20737,8 @@
      }
     }
    },
-   "v1.RBDVolumeSource": {
-    "id": "v1.RBDVolumeSource",
+   "v1.RBDPersistentVolumeSource": {
+    "id": "v1.RBDPersistentVolumeSource",
```

Comment:
@rootfs is this a backward compatible change ?

Reply:
It is wire compatible. The type name never appears in serialized proto and json, and the new struct contains identical fields, with the addition of a namespace field in the secret ref